zrml/prediction-markets/src/tests.rs

```diff
@@ -1345,13 +1345,9 @@ fn create_market_and_deploy_assets_results_in_expected_balances() {
     let metadata = gen_metadata(42);
     let category_count = 4;
     let assets = MarketType::Categorical(category_count);
-    let extra_amount = 10 * BASE;
-    let min_liqudity = <Runtime as zrml_swaps::Config>::MinLiquidity::get();
-    let amount = min_liqudity + 2 * extra_amount;
-    let weights = vec![<Runtime as zrml_swaps::Config>::MinWeight::get(); 5];
-    let amount_base_asset = amount;
-    let amounts = vec![amount - extra_amount, amount, amount, amount];
+    let amount = 123 * BASE;
     let pool_id = 0;
+    let weights = vec![2 * BASE; 5];
```

Comment:
This test fails once `MinWeight` changes to something greater than `2 * BASE` in primitives crate.

Reply:
So will any test that uses `create_initial_pool`. We could change `2 * BASE` to `MinWeight`, but you could still argue that the test will fail if we change the minimum of maximum number of assets in the pool below or above 5. Or the min/max category count.

Personally, I don't think it's necessary that we future proof our tests against any possible change of constants. I guess I could replace `MinWeight` with a mock-specific weight. But if we start there, where do we stop?

Reply:
I see, it's difficult to draw the line there. But it's also inconsistent in the current state, for example for `MinLiquidity` we use the constant instead of `1_000_000_000_000` that is supplied by the runtime. Generally I would use constants specified within the test runtime. The value `5` is arbitrary for that test and not a global constant within that pallet.

What I also realized is that it might be better for the reader to name the values instead of using raw numbers, in cases where numbers repeatedly used a constant is appropriate. But that's something for another meeting I guess - Now that we're plenty of people with different styles, I guess it would be good to have some guidelines that we try to adhere to.